settings/server/methods.js

```
@@ -78,8 +78,8 @@ Meteor.methods({
     } catch (error) {
       // otherwise show an error
       const message = `Update gitHub configuration: ${error}`;
-
-      }
+      console.log(message);
```

Instead of `console.log`, we should throw an error. Something like: `return new Meteor.Error(message);`